docusaurus.config.js

```diff
@@ -104,35 +104,52 @@ const config = {
                 label: 'About TCET Open Source',
                 to: '/docs/about-tcetopensource',
               },
+              {
+                label: 'Documentation Website',
+                to: '/docs/projects/docs-site/about-docs',
+              },
+              {
+                label: 'TCET Linux',
+                to: '/docs/projects/tcet-linux/about-tcet-linux',
+              }
             ],
           },
-          // {
-          //   title: 'Community',
-          //   items: [
-          //     {
-          //       label: 'Stack Overflow',
-          //       href: 'https://stackoverflow.com/questions/tagged/docusaurus',
-          //     },
-          //     {
-          //       label: 'Discord',
-          //       href: 'https://discordapp.com/invite/docusaurus',
-          //     },
-          //     {
-          //       label: 'Twitter',
-          //       href: 'https://twitter.com/docusaurus',
-          //     },
-          //   ],
-          // },
           {
             title: 'More',
             items: [
               {
                 label: 'Blog',
                 to: '/blog',
               },
+            ],
+          },
+          {
+            title: 'Community',
+            items: [
+              {
+                label: 'Github',
+                to: 'https://github.com/tcet-opensource/documentation',
+              },
+              {
+                label: 'LinkedIn',
+                to: 'https://www.linkedin.com/company/tcet-opensource/mycompany/',
+              },
+              {
+                label: 'Discord',
+                to: 'https://t.co/bov6iUrZcB',
```

Comment:
Replace the Twitter embed URL with official Discord invite.

Reply:
Twitter Embed URL is redirecting to twitter you want the url to redirect to discord?